package.json

```diff
@@ -62,7 +62,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged --pattern 'packages/**/src/**/*' && npx tslint -c tslint.json --project ."
+      "pre-commit": "pretty-quick --staged --pattern 'packages/**/src/**/*' && npx eslint . --ext .ts,.tsx"
```

Comment:
I don't think we have `.tsx` in this project, also it should be same as `npm run lint` https://github.com/cennznet/api.js/pull/159/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19

Reply:
done